### ISSUE TITLE
progress crashes

### ISSUE THREAD

**Issue by treinen** [colis-anr/shstats#20]
create a directory "corpus" containing one preinst script. Then execute twice the command "./run corpus". The second time it crashes with message

````
Fatal error: exception File "progress.ml", line 57, characters 2-8: Assertion failed
````

If you remove after the first run the file corpus/preinst.shstats-cache then it does not crash.

**Comment by Niols** [comment]
Hum, I cannot reproduce this on master, can you?

**Comment by treinen** [comment]
Yes. I can.  One really has to use a corpus directory with only one or two scripts, and use the --cache option to bin/shstats.

**Comment by Niols** [comment]
Indeed, the `--cache` options was important to reproduce and wasn't in my `./run` script. This should be closed by a21f046, can you confirm?

**Comment by treinen** [comment]
Yes, confirmed. Thanks -Ralf.
